Sources/Markdown/Base/RawMarkup.swift

```diff
@@ -24,7 +24,7 @@ enum RawMarkupData: Equatable {
     case thematicBreak
     case htmlBlock(String)
     case listItem(checkbox: Checkbox?)
-    case orderedList
+    case orderedList(start: UInt = 1)
```

Comment:
thoughts on calling this `startIndex` throughout to make it clearer what 'start' we're referring to?

Reply:
Sounds good, i've pushed this change up.